authappliance/pi-appliance

```diff
@@ -157,8 +157,7 @@ class WebserverMenu(object):
 
 class Peer(object):
 
-    files = ["/etc/privacyidea/enckey", "/etc/privacyidea/pi.cfg",
-             "/etc/privacyidea/logging.cfg",
+    files = ["/etc/privacyidea/enckey",
```

Comment:
Why don't you replicate pi.cfg and logging.cfg (if logging exists)

The pi.cfg might contain the SUPERUSER_REALMS. If this is not copied to the remote host, the remote host will not work properly.